File: Python/Iniciante/Solucao_1001.py
try:#Try para tratamento de erro
    #as variaveis a e b recebem um entrada do tipo int, variavel x receber a soma de a + b
    a = int(input())
    b = int(input())
    x = a + b
    print(f"X = {x}")#print para variavel x
except ValueError:
    exit()

File: Java/Iniciante/Solucao_1001.java
import java.io.IOException;
import java.util.Scanner;

public class Solucao_1001 {
    public static void main(String[] args) throws IOException{
        var input = new Scanner(System.in);

        int a, b, x;
        a = input.nextInt();//entrada de a e b
        b = input.nextInt();
        x = a + b;//Variavel X receber a soma de a + b

        System.out.printf("X = %d%n",x);//Print de X no estilo de C %n pula linha

        input.close();//fechamento do input
    }
}
